packages/openapi-v2/test/unit/controller-spec/controller-decorators.test.ts

```diff
@@ -74,6 +74,7 @@ describe('Routing metadata', () => {
           },
         },
       },
+      definitions: {},
```

Comment:
Please revert these changes - see https://github.com/strongloop/loopback-next/pull/860/files#r161257117

Alternatively, rework the test suite in this file to be more robust:

 - modify the tests where you had to add an empty `definitions` property in such way that they don't depend on exact shape of the spec object (e.g. assert only the value of `paths` property)
 - ensure there is a single test verifying the top-level properties of the object returned by `getControllerSpec`